dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -70,6 +70,7 @@ def self.statuses
     before_save :log_status, if: -> {status_changed?}
 
     def should_send_decision_email?
+      return false if regional_partner&.applications_decision_emails == RegionalPartner::SENT_BY_PARTNER
```

Comment:
It's a little counterintuitive for me to read this - I see the statement after the return and it seems redundant

Suggest
```
if regional_partner&.applications_decision_emails == RegionalPartner::SENT_BY_PARTNER
  false
else
  AUTO_EMAIL_STATUSES.include?(status)
end
```